server/index.js

```diff
@@ -35,6 +36,10 @@ app.use(RequestSyntaxErrorHandler);
 
 startServer();
 
+// ----- Start cron tasks
+removeDuelsTask.start();
+checkDuelsTask.start();
```

Comment:
I think these should go in `startServer()`, once the app is ready, because when I start the server for the first time, I have these errors

<details>
<summary>Logs</summary>
<pre>
error: Can't check duels Error: ER_BAD_FIELD_ERROR: Unknown column 're_last_time' in 'field list'
    at Query.Sequence._packetToError ([...]/guacamole/server/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket ([...]/guacamole/server/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket ([...]/guacamole/server/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket ([...]/guacamole/server/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write ([...]/guacamole/server/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write ([...]/guacamole/server/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> ([...]/guacamole/server/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> ([...]/guacamole/server/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (node:events:378:20)
    at addChunk (node:internal/streams/readable:313:12)
    --------------------
    at Protocol._enqueue ([...]/guacamole/server/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Connection.query ([...]/guacamole/server/node_modules/mysql/lib/Connection.js:198:25)
    at file://[...]/guacamole/server/db/database.js:157:16
    at new Promise (<anonymous>)
    at queryPromise (file://[...]/guacamole/server/db/database.js:156:10)
    at Task.execution (file://[...]/guacamole/server/files-script/cron-tasks.js:38:23)
    at [...]/guacamole/server/node_modules/node-cron/src/task.js:52:23
    at new Promise (<anonymous>)
    at Task.update ([...]/guacamole/server/node_modules/node-cron/src/task.js:50:7)
    at Timeout.ScheduledTask.task [as _onTimeout] ([...]/guacamole/server/node_modules/node-cron/src/scheduled-task.js:86:10) {
  code: 'ER_BAD_FIELD_ERROR',
  errno: 1054,
  sqlMessage: "Unknown column 're_last_time' in 'field list'",
  sqlState: '42S22',
  index: 0,
  sql: 'SELECT duel.du_id, us_login, re_answers, re_last_time                FROM results, duel                WHERE results.du_id = duel.du_id                  AND duel.du_inProgress = 1;'
}
</pre>
</details>